crates/node-core/src/version.rs

```diff
@@ -11,6 +15,39 @@ pub const VERGEN_GIT_SHA: &str = env!("VERGEN_GIT_SHA");
 /// The build timestamp.
 pub const VERGEN_BUILD_TIMESTAMP: &str = env!("VERGEN_BUILD_TIMESTAMP");
 
+//The identification of the client
+/// - The two letter client code
+/// - The human readable name of the client
+/// - The version string of the current implementation 
+/// - first four bytes of the latest commit hash of this build
+/// 
+/// # Example
+/// 
+///  ```text
+// {
+//   code: "RH",
+//   name: "Reth",
+//   version: "v0.2.0-beta.5",
+//   commit: "defa64b2"
+// }
+///  ```
+struct ClientVersionV1<'a> {
+    code: &'a str,
+    name: &'a str,
+    version: &'a str,
+    commit: &'a str,
+}
+
+pub const CLIENTVERSIONV1: ClientVersionV1<'_> = ClientVersionV1 {
+    code: env!("CLIENT_CODE"),
+    name: env!("NAME_CLIENT"),
```

Comment:
these are statics, so this should be

```suggestion
    code: CLIENT_CODE,
    name: NAME_CLIENT,
```